facade/dfs.go

```diff
@@ -646,3 +646,31 @@ func (info *registryVersionInfo) start(isvcsRoot string, hostPort string) (*dock
 func (f *Facade) DockerOverride(ctx datastore.Context, newImageName, oldImageName string) error {
 	return f.dfs.Override(newImageName, oldImageName)
 }
+
+// Interface to allow filtering DFS clients
+type DfsClientValidator interface {
+	ValidateClient(string) bool
+}
+
+type clientValidator struct {
+	context datastore.Context
+	facade  *Facade
+}
+
+func NewDfsClientValidator(fac *Facade, ctx datastore.Context) DfsClientValidator {
+	return &clientValidator{ctx, fac}
+}
+
+func (val *clientValidator) ValidateClient(hostIP string) bool {
+	host, err := val.facade.GetHostByIP(val.context, hostIP)
+	if err != nil || host == nil {
+		glog.Errorf("Unable to load host with ip %s", hostIP)
```

Comment:
Should this be an Error? Error means we get a support ticket and the operator gets paged. Warning, maybe?